app/components/address_component.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class AddressComponent < ViewComponent::Base
+  def initialize(address, raw_location: nil)
+    @address = address
+    @raw_location = raw_location
+  end
+
+  def formatted_address
+    # rubocop:disable Rails/OutputSafety
```

Comment:
this also seems a good thing to remove

Reply:
Will give it a go and see 🙌🏿

Reply:
Resolved in [9b10173](https://github.com/geeksforsocialchange/PlaceCal/pull/2638/commits/9b1017332dcc523fbb76326bca1dec6ad9ba321f)